numba/targets/cpu.py

```diff
@@ -128,8 +128,8 @@ def build_set(self, builder, set_type, items):
         return setobj.build_set(self, builder, set_type, items)
 
     def post_lowering(self, mod, library):
-        if self.enable_fastmath:
-            fastmathpass.rewrite_module(mod)
+        if self.fastmath:
+            fastmathpass.rewrite_module(mod, self.fastmath)
```

Comment:
There are cases where `self.fastmath` arrives here as a boolean type which I think is the root cause of all the test failures in CI.